### ISSUE TITLE
Integrate an auto attack/shot bar

### ISSUE THREAD

**Issue by Cruzix** [shagu/pfUI#244]
Integrate an auto attack/shot bar. I would highly prefer if you could make the auto SHOT (ranged) bar have similar visual presentation as Rais_AutoShot as it's far better than any other addons' way of doing it.

Here's how Rais_AutoShot looks: 
![gif](https://cloud.githubusercontent.com/assets/6162744/21522087/eb95215a-cd02-11e6-8c35-391bd6a6094d.gif)

White bar is the "cooldown" time of your auto shot
Red bar is the cast time of your auto shot

**Comment by shagu** [comment]
Class specific stuff will be started after Release 1.0 is pushed.

**Comment by Road-block** [comment]
Hell, that AttackBar addon code is atrocious.

**Comment by Cephel** [comment]
https://github.com/Terrub/Samuel

This one works pretty well and could be integrated into pfUI quite easily

**Comment by Cruzix** [comment]
@Cephel that would be a nice alternative to abar indeed (melee), not for rais though (ranged).

**Comment by shagu** [comment]
> Isn't YaHT a better version of rais?

Yes, but his comment is from Dec/2016 and YaHT was released on February/2017.
